flaskr.egg-info/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
```

Comment:
could you add all of the files in this folder to .gitignore? This stuff gets built whenever app is launched and it does not need to be a part of the repository.